File: Step03_Arrays/Lec1_Easy/Program_01.java
package Step03_Arrays.Lec1_Easy;

import java.util.*;

// Largest Element in an Array

public class Program_01 {
    public static void main(String[] args) {
        int[] nums = {7, 8, 66, 4, 2, 4, 6, 88, 2};
        
        // processing
        int largest = nums[0];
        for (var element : nums) {
            if (element > largest) {
                largest = element;
            }
        }

        System.out.println("Largest Element : " + largest);
    }
}

// Time-complexity : O(N)

File: Step03_Arrays/Lec1_Easy/Program_01.cpp
#include <bits/stdc++.h>
using namespace std;

// globally => maxArrSize: 10^7
// locally  => maxArrSize: 10^6

// Largest Element in an Array

int main() {
    int n;
    cin >> n;
    vector<int> arr;
    for (int i = 0; i < n; i++) {
        int x;
        cin >> x;
        arr.push_back(x);
    }

    // processing
    int largest = arr[0];
    for (auto element : arr) {
        if (element > largest) {
            largest = element;
        }
    }

    cout << "Largest element : " << largest;
}

// Time-complexity : O(N)